Front-end-Projects/Basic/Starbucks-Landing-page/README.md

```diff
@@ -0,0 +1,63 @@
+<h1 align='center'><b>💥Starbucks Landing Page💥</b></h1>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h3 align='center'>Tech Stack Used 🎮</h3>
+
+
+<div align='center'>
+
+  ![HTML5](https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white)
+  ![CSS3](https://img.shields.io/badge/css3-%231572B6.svg?style=for-the-badge&logo=css3&logoColor=white)
+
+</div>
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Description 📃
+
+<div>
+  <!-- <p>Add Description of the project</p> -->
+    <p>This project is a simple HTML and CSS implementation of a Starbucks website clone. It aims to replicate the basic structure and styling of the popular website Starbucks. This project Showcases the power of HTML and CSS.</p>
+</div>
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: How to run it? 🕹️
+
+To run this project locally, follow these steps:
+
+1. Fork the repository.
+
+2. Clone the repository to your local machine:
+git clone https://github.com/your-username/Starbucks-Landing-page.git
+
+3. Open the project folder in your preferred code editor, now you can view website in live.
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+
+<img src="Starbucks-Landing-Page.webp">
```

Comment:
![image](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/b7cf2c5e-4029-44fe-b77f-e4982abbca1b)
Image is not shown. Try '/Starbucks-Landing-Page.webp' OK.
And delete Startbucks.webp file. Only 1 screenshot is sufficient.